server/tests/test_chatroom.py

```diff
@@ -0,0 +1,205 @@
+from server import chatroom
+import random
+import string
+import datetime
+import json
+
+def test_create_chatroom(postgresql):
+    with open('database/create_tables.sql', 'r') as sqlfile:
+        cur = postgresql.cursor()
+        cur.execute(sqlfile.read())
+    cur = postgresql.cursor()
+    cur.execute("INSERT INTO student(email, password, salt, student_id) VALUES('a', 'a', 'a', '123')")
+    cur.execute("INSERT INTO university values('NYU')")
+    cur.execute("SELECT * FROM chatroom")
+    invite_id = ''.join(random.choices(string.ascii_uppercase + string.digits, k=10))
+    chatroom_name = 'a'
+    uni_id = 'NYU'
+    cur.execute("INSERT INTO chatroom(chatroom_name, invite_id, uni_id) VALUES(%(chatroom_name)s, %(invite_id)s, %(uni_id)s)", {"chatroom_name" : chatroom_name, "invite_id" : invite_id, "uni_id" : uni_id})
+    
+    cur.execute("SELECT id FROM chatroom where chatroom_name = %(chatroom_name)s   AND uni_id = %(uni_id)s", {"chatroom_name" : chatroom_name, "uni_id" : uni_id})
+    chatroom_id = cur.fetchall()[0][0]
+
+    user_id = '123'
+    cur.execute(
+            "INSERT INTO in_chatroom(student_id, uni_id, chatroom_id) VALUES(%(user_id)s, %(uni_id)s, %(chatroom_id)s)",
+            {"user_id": user_id, "uni_id": uni_id, "chatroom_id": chatroom_id})
+    #cur.execute("SELECT * FROM chatroom")
+    """
+    cur.execute("SELECT * FROM chatroom")
+    chatroom_name = cur.fetchall()[0][0]
+    cur.execute("SELECT invite_id FROM chatroom")
+    print(cur.fetchall()[0][0])
+    """
+    
+    cur.execute("SELECT student_id FROM in_chatroom")
+    s_id = str(cur.fetchall()[0][0])
+
+    assert(s_id == user_id)
+
+def test_create_chatroom_func(postgresql):
```

Comment:
why are u having two create chatroom tests? cant we just hv this one that calls the function? we only need to make sure the function works so its purpose is a little bit unclear to me